src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java

```diff
@@ -238,7 +238,7 @@ public void testGenotypesOnly(File input, File expected, List<String> extraArgs,
     }
 
     //this only tests single-sample
-    @Test(dataProvider = "getGVCFsForGenomicsDB", timeOut = 1000)
+    @Test(dataProvider = "getGVCFsForGenomicsDB", timeOut = 1000000)
```

Comment:
Whoops! Looks like I gave you a poor choice of number. I forgot to account for the runtime units being in terms of minutes...